.github/workflows/lock-versions.yml

```diff
@@ -6,7 +6,7 @@ on:
           description: 'List of branches to lock versions (ordered, comma separated)'
           required: true
           # 7.17 was intentionally skipped because it was added late and was bug fix only
-          default: '7.13,7.14,7.15,7.16,8.0,8.1,8.2'
+          default: '7.13,7.14,7.15,7.16,8.0,8.1,8.2,8.3'
```

Comment:
@brokensound77 do we want to adjust this as well based on the approved backport versions changed in `backport.yml`?